templates/back-office/crubside-arrive-content.hypr

```diff
@@ -0,0 +1,42 @@
+<div class="curbside-page-content">
+    <div>
+        <p>{{labels.curbsideSeeYouSoon}}</p>
+        <p>{{ labels.curbsideShipmentText }} </p>
+    </div>
+
+    <div class="curbside-spacing">
+        <button class="mz-delivery-button mz-button mz-button-large" onclick="location.href = 'http://{{domainName}}/anonymous-notification/curbsideInfo/{{model.shipmentNumber}}/{{model.orderId}}';"> {{ labels.CurbSidebtnText }}  </button>
+      
+    </div>
+
+    <div class="curbside-notedetails"> 
+        <p>{{labels.curbsideNoteTitle}}</p>
+        <ul>
+        <li>{{labels.curbsideIdText}}</li>
+        <li>{{labels.curbsideRefundText}}</li>
+        <li>{{labels.curbsideLandingText}}</li>
+        </ul>
+    </div>
+
+    <div class="curbside-order-details">
+        <div class="curbside-order-details-left">
+         <h2  style="color:white"> <b>{{labels.orderDetails}}</b></h2>
```

Comment:
Other than perhaps emails please do not use inlines styles. 

Our customers heavily customize these templates and it is easier to not have to find and remove inline styles.